src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -118,10 +125,14 @@ internal class TaskReceiver(tasksDirectory: String,
         }
 
         val md5sum = dockerProcessor.getMd5Sum(rootFile)
-        this.callback("md5 sum of root file", md5sum)
+        this.onMessageProcessed("md5 sum of root file", md5sum)
+
+        val onPublishFailureCallback = {
+            LOGGER.info("Publish failed: taskId = $taskId, status code: ${StatusCode.SUCCESS}, md5 sum: $md5sum")
```

Comment:
I can't help crying when looking at the failure message which reports success status code :) It appears that you just don't need it here at the moment, do you?

Reply:
Apparently not :)
Is it okay, that I changed to empty callback?
When there will be a status code, the callback will be more useful.

Reply:
Writing a message ok. Writing a little bit schizophrenic message is a little bit not ok.

Reply:
Yeah, this message looks too verbose.